### ISSUE TITLE
Discussion - 2015-11-13 post-speech

### ISSUE THREAD

**Issue by kennybytes** [scel-hawaii/intro-presentation#4]
@cobatake 

This is a place to discuss some of the good things about the recruitment talk on 2015-11-13 and some of the shortcomings of that talk... I wanted to try this as a possible alternative to meeting in person. Discussion in this thread will eventually be archived on the scel wiki when we're done. 

Here are some of my comments:
- We might need to figure out where to split the talks
- The talk overall was pretty rough - but it's okay, we're new at this sort of talk
- Needs more technical details
- Poorly organized in some parts
- Venue could have been better

I feel there definitely needs more work to improve this particular presentation - it's still very rough.

**Comment by cobatake** [comment]
@FFAxKenny 

My comments are pretty similar:
- We need to figure out who is covering which slides
- Could make the slides more interactive so students can participate
- Need to add more details to some parts
- Could use a motivation slide
- Check slides better the next time for errors

I know the talk was rough, but I think the dinner part after was good.  We could maybe have other students from the lab there to interact with them too in the future.

**Comment by kennybytes** [comment]
Yeah, the dinner part was good - we also need to send a follow up email. Do you want to draft one up?

**Comment by cobatake** [comment]
I'll draft one up and send it to you tonight.
